src/components/team/InviteMembers.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import Popup from "reactjs-popup";
+import useAxios from "../../common/hooks/useAxios";
+import { isErrorResponse } from "../../interfaces/ErrorResponse";
+import { IPlayer } from "../../interfaces/IPlayer";
+
+// const tempPlayerList = [
+//     {
+//         id: "fff1",
+//         firstName: "Noah",
+//         lastName: "Roerig",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+//     {
+//         id: "fff2",
+//         firstName: "Stevan",
+//         lastName: "Perrino",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+//     {
+//         id: "fff3",
+//         firstName: "Jimmy",
+//         lastName: "Cannon",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+//     {
+//         id: "fff4",
+//         firstName: "Jacob",
+//         lastName: "Hropoff",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+// ];
+
+interface IInviteMembers {
+    teamId: number;
+}
+
+function InviteMembers({ teamId }: IInviteMembers) {
+    const [isLocked, setIsLocked] = useState(false);
+    const [option, setOption] = useState<string>("");
+    const [players, setPlayers] = useState<IPlayer[]>();
+
+    const { getRequest, postRequest } = useAxios();
+
+    useEffect(() => {
+        if (players && players.length > 0) {
+            setOption(players[0].authId);
+        }
+    }, [players]);
+
+    // idea here is to pause just slightly before searching so the user can type in some more
+    // words to find there person. Just so there isn't a api call every single letter
+    const searchForUsers = (value: string) => {
+        if (isLocked) {
+            return;
+        }
+        setIsLocked(true);
+        setTimeout(async () => {
+            // search for users using api
+            const concatenatedValue = value.toLowerCase().trim().replace(/\s/g, "");
+            const response = await getRequest<IPlayer[]>(
+                `/players/search?name=${concatenatedValue}`
+            );
+            if (isErrorResponse(response)) {
+                setIsLocked(false);
+                return;
+            }
+            setPlayers(response);
+            setIsLocked(false);
+        }, 700);
+    };
+
+    const inviteUser = async (authId: string) => {
+        const response = await postRequest(`/players/${authId}/requests/teams/${teamId}`);
+        if (isErrorResponse(response)) {
+            return;
+        }
+
+        console.log("invite sent");
+    };
+
+    return (
+        // <Popup trigger={<button>Invite Members</button>} modal>
+        <div style={{ backgroundColor: "grey" }}>
```

Comment:
With this whole input then submitting setup it is basically a form, but problem is that it isn't wrapped in a form tag, so the browser can't be extra fancy and smart and the only way to submit the form is by focusing the button and hitting enter or by click the button but if you wrap this in a form you can handle the submission of the form and have extra way to submit it by hitting enter after making the selection this will also help with accessibility. But handling the form you will need to preventDefault